tests/bootstrap.php

```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types = 1);
```

Comment:
Code style checks are running on `/tests` so that's why this is fixed. Should we also cover `/config` ?